src/frontend/components/Felleskomponenter/Arbeidsperiode/Arbeidsperiode.tsx

```diff
@@ -57,7 +53,9 @@ export const Arbeidsperiode: React.FC<Props> = ({
 }) => {
     const { tekster } = useApp();
     const { erÅpen: arbeidsmodalErÅpen, toggleModal: toggleArbeidsmodal } = useModal();
-    const barnetsNavn = !!barn && barn.navn;
+    const teksterForModal: IArbeidsperiodeTekstinnhold =
+        tekster().FELLES.modaler.arbeidsperiode[personType];
+    const { flerePerioder, leggTilKnapp } = teksterForModal;
```

Comment:
Kunne dette vært gjort på en linje? 
```suggestion
    const { flerePerioder, leggTilKnapp } = 
        tekster().FELLES.modaler.arbeidsperiode[personType];
```